cfg/dev.js

```diff
@@ -1,12 +1,8 @@
-const path                 = require( 'path' );
+const baseConfig = require( './base' );
 
-const { merge }            = require( 'lodash' );
-
-const baseConfig           = require( './base' );
-
-
-const devConfig = merge( {}, baseConfig, {
-    entry  : path.join( __dirname, '../src/index.js' ),
+module.exports = baseConfig( {
+    mode   : 'development',
+    inline : false,
```

Comment:
We need to inline the styles for LochNess
```suggestion
    inline : true,
```